soccer/src/soccer/soccer_mom.cpp

```diff
@@ -0,0 +1,35 @@
+#include <soccer_mom.hpp>
+
+namespace tutorial {
+Soccer_Mom::Soccer_Mom() : Node("soccer_mom") {
+    publisher_ = create_publisher<std_msgs::msg::String>("team_fruit", 10);
+    subscription_ = create_subscription<rj_msgs::msg::TeamColor>(
+        referee::topics::kTeamColorTopic, rclcpp::QoS(1).transient_local(),
+        [this](rj_msgs::msg::TeamColor::SharedPtr color) {  // NOLINT
+            // RCLCPP_INFO(this->get_logger(), "ASDF %s", color->is_blue);
+            recieve(color->is_blue);
+        });
+    timer_ = create_wall_timer(std::chrono::milliseconds(16), [this]() { send(); });
+    // is_blue_ = true;
+}
+
+void Soccer_Mom::recieve(bool b) {
+    // if (b == is_blue_) {
+    //     return;
+    // }
+    is_blue_ = b;
```

Comment:
because this function is so short you can just write it inside the lambda; but that's a style choice not necessarily incorrect